src/Simulation/Simulators.Tests/StackTraceTests.cs

```diff
@@ -29,32 +31,33 @@ public StackTraceTests(ITestOutputHelper output)
         [Fact]
         public void AllocateQubit2Test()
         {
-            using (var sim = new QuantumSimulator())
+            using var sim = new QuantumSimulator();
```

Comment:
NIT: I see a mix of style between using and not parenthesis. Was this intentional?

Reply:
Parenthesis aren't allowed in the new C# 8 form that uses the current scope to trigger disposability. For more details: https://docs.microsoft.com/en-us/dotnet/csharp/whats-new/csharp-8#using-declarations